### PR TITLE
Init app

### PR DESCRIPTION
**Pull request by thayannevls**
#### What is the purpose of this pull request?
Init app and sent `installWebApp` event to google tag manager `dataLayer`.

#### What problem is this solving?
We need to measure and show how much PWA features contributed to the overall user experience on VTEX stores.

#### How should this be manually tested?
[Workspace](https://pwaevents2--storecomponents.myvtex.com/checkout/orderPlaced/?og=949500478939)
1. Click 'install' on install banner at the bottom of the page
2. Choose the option you desire on the A2HS prompt
3. Open the console, type `dataLayer` and check if the event was added

#### Types of changes

* [ ] Bug fix (a non-breaking change which fixes an issue)
* [x] New feature (a non-breaking change which adds functionality)
* [ ] Breaking change (fix or feature that would cause existing functionality to change)
* [ ] Requires change to documentation, which has been updated accordingly.

### CODE REVIEW

File: public/metadata/en-US/full_description.txt [hunk 1]
```diff
@@ -0,0 +1,12 @@
+## About
+
+PWA Google Tag Manager Pixel is an extension of the [Google Tag Manager App](https://github.com/vtex-apps/google-tag-manager) for tracking the activities PWA related of visitors to your ecommerce.
```
Comment (lucasecdb) [inline public/metadata/en-US/full_description.txt]:
```suggestion
PWA Google Tag Manager Pixel is an extension of the [Google Tag Manager App](https://github.com/vtex-apps/google-tag-manager) for tracking the activities PWA related of visitors to your e-commerce.
```

File: public/metadata/pt-BR/full_description.txt [hunk 2]
```diff
@@ -0,0 +1,12 @@
+# Sobre 
+
+PWA Google Tag Manager é uma extensão da [Google Tag Manager App](https://github.com/vtex-apps/google-tag-manager) para rastreamento das atividades relacionadas a PWA dos visitantes de seu ecommerce. Ferramenta usada para avaliar a eficiência de seus anúncios.
```
Comment (lucasecdb) [inline public/metadata/pt-BR/full_description.txt]:
```suggestion
PWA Google Tag Manager é uma extensão da [Google Tag Manager App](https://github.com/vtex-apps/google-tag-manager) para rastreamento das atividades relacionadas a PWA dos visitantes de seu e-commerce. Ferramenta usada para avaliar a eficiência de seus anúncios.
```

File: pixel/head.html [hunk 3]
```diff
@@ -1,11 +1,5 @@
 <script>
   (function() {
-    var appId = "{{settings.gtmId}}";
-    if (!appId) {
-      console.error('Warning: No APP NAME ID is defined. Please configure it in the apps admin.');
-      return
-    }
-
-    // Add script here
+    // the script is setup in google-tag-manager app
```
Comment (klzns) [inline pixel/head.html]: You can delete this file

File: manifest.json [hunk 4]
```diff
@@ -1,9 +1,9 @@
...
+  "name": "pwa-gtm-pixel",
   "vendor": "vtex",
   "version": "0.0.1",
-  "title": "CHANGEME",
-  "description": "CHANGEME",
+  "title": "PWA Google Tag Manager Pixel",
+  "description": "VTEX App that integrates PWA events with Google-tag-manager",
   "billingOptions": {
     "termsURL": "",
     "support": {
```
Comment (klzns) [inline manifest.json:9]: You can remove pixel builder, since the folder pixel will be empty

File: react/typings/events.d.ts [hunk 5]
```diff
@@ -107,6 +108,13 @@ export interface ProductImpressionData extends EventData {
   list: string
 }
 
+export interface InstallWebAppData extends EventData {
+  event: 'installWebApp',
+  eventName: 'vtex:installWebApp'
+  userChoice: string
```
Comment (klzns) [inline react/typings/events.d.ts]: Is that an enum?

**Review by klzns — CHANGES_REQUESTED** [review]

File: manifest.json (hunk 4 above)
Comment (thayannevls) [reply manifest.json:9]: isn't the pixel builder necessary for the configuration on `plugins.json` and `interfaces.json`?

File: react/typings/events.d.ts (hunk 5 above)
Comment (thayannevls) [reply react/typings/events.d.ts]: You're right, just changed it!

File: manifest.json (hunk 4 above)
Comment (klzns) [reply manifest.json:9]: nope, not related

**Review by lucasecdb — APPROVED** [review]
lgtm

**Review by klzns — APPROVED** [review]

**Pull request merged**